apps/prairielearn/src/pages/administratorAdmins/administratorAdmins.html.ts

```diff
@@ -46,8 +46,6 @@ export function AdministratorAdmins({
                     id: 'administratorInsertButton',
                   }),
                 )}"
-                data-trigger="manual"
-                onclick="$(this).popover('show')"
```

Comment:
This is the default behavior; I don't know why this is here. Maybe it's just been copy/pasted around forever? I removed it from all admin pages and a few others too.